LSP-metals.sublime-settings

```diff
@@ -13,6 +20,30 @@
     "-Xms100m"
   ],
 
-  // Optional path to the Java home directory. Requires reloading sublime.
-  "java_home": ""
+  // The startup command (ST4 only)
+  "command": [
+    "${java_path}",
+    "-Dmetals.client=sublime",
+    "-Dmetals.status-bar=on",
+    "-Dmetals.execute-client-command=on",
+    "-Xss4m",
+    "-Xms100m",
+    "-jar",
+    "${packages}/LSP-metals/coursier",
+    "launch",
+    "--ttl",
+    "Inf",
+    "--repository",
+    "bintray:scalacenter/releases",
+    "--repository",
+    "sonatype:snapshots",
+    "--main-class",
+    "scala.meta.metals.Main",
+    "org.scalameta:metals_2.12:${server_version}"
+  ],
+
+  // initialization options (ST4 only)
+  "initializationOptions": {
```

Comment:
in this PR it is used only for ST3, `initializationOptions` should be set in both versions